### DISCUSSION TITLE
Definable color of y axis on graph

### DISCUSSION THREAD

**Discussion by Jusufs** [grafana/grafana#44527] [Dashboards, Panels & Visualizations]
If the color of each y axis are parametrisable, we can see on first look , which curve is corresponding (same color) to this axis.
Thanks.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
